src/logging/monitor.ts

```diff
@@ -96,15 +139,25 @@ class LogMonitor {
 							self.containers[row.CONTAINER_ID_FULL]
 						) {
 							await self.handleRow(row);
+						} else if (self.HOST_SERVICES.includes(row._SYSTEMD_UNIT)) {
+							await self.handleHostServiceRow(row, logSystemMessage);
+						} else if (isSupervisorStartupMessage(row)) {
+							// Log Supervisor startup message, but in chronological order,
+							// allowing any earlier timestamped logs to be processed first.
```

Comment:
Did you get to the bottom of the problem? How is it that before we could include logs from before the supervisor start when we stored the logs on the database? Is it something on the backend that changed that does not allow sending logs out of order?

I think we can test this by sending some logs with `curl` to the backend and playing with timestamps

Reply:
My deduction of the behavior I observed being due to the backend not accepting out-of-order logs was an incorrect deduction. I suspect now that there was something weird in the way I set up the system log-streaming, or the use of `logSystemMessage` when calling `logMonitor.start`.